lib/spanner/command/request.ex

```diff
@@ -48,10 +48,15 @@ defmodule Spanner.Command.Request do
 
   defp read_config(request, config_path) do
     [bundle, _cmd] = String.split(request.command, ":", parts: 2)
-    cmd_config_file = Path.join([config_path, bundle, Config.dynamic_file_name()])
-    case File.exists?(cmd_config_file) do
+    cmd_config_dir = Path.join([config_path, bundle])
+
+    case File.dir?(cmd_config_dir) do
       true ->
-        Config.Parser.read_from_file(cmd_config_file)
+        with {:ok, files} <- File.ls(cmd_config_dir),
+             cmd_config_file when is_binary(cmd_config_file) <- Enum.find(files, {:ok, ""}, &Spanner.Config.config_file?/1) do
```

Comment:
Yeah I like the `:not_found` symbol. I was pretty confused by what `{:ok, ""}` meant.